### DISCUSSION TITLE
Attention Grabber additional feature

### DISCUSSION THREAD

**Discussion by gahara42** [adieyal/sd-dynamic-prompts#318] [Ideas]
Hello! I'm not sure if this is right place to ask for this, i wanted to make an issue first, but decided that it's not really a groundbreaking bug.

I would like to ask about adding a check mark in attention grabber to ignore Lora's syntax, because most of the time you want Lora weight at fixed value.

Current behavior of this feature trying to add attention to Lora in prompt in different variations.

Examples:

original prompt is `<lora:loraname:0.7>`

> when using attention grabber

`<(lora:1.42):loraname:0.7>`
`<lora:(loraname:0.7:1.1)>`

It would be great to have option for attention grabber to ignore Lora's.

**Comment by adieyal** [comment]
Created a new feature here: https://github.com/adieyal/sd-dynamic-prompts/issues/327

**Reply by gahara42** [reply]
> Created a new feature here: #327

Thank you! Really appreciate your work!
